roles/teuthology/tasks/setup_users.yml

```diff
@@ -8,6 +8,34 @@
   tags:
     - user
 
+# tkill group gets sudo rights to /bin/kill for when teuthology-kill calls it
+- name: Create tkill group
+  group:
+    name: tkill
+    state: present
+  tags:
+    - user
+
+- name: Add tkill_users to tkill group
+  user:
+    name: "{{ item }}"
+    groups: tkill
+    append: yes
+  with_items: "{{ tkill_users }}"
+  tags:
+    - user
+  when: tkill_users is defined and tkill_users|length > 0
+
+- name: Grant tkill_users sudo access to /bin/kill
+  lineinfile:
+    dest: /etc/sudoers.d/cephlab_sudo
+    regexp: "^%tkill"
+    line: "%tkill ALL=NOPASSWD: /bin/kill, /usr/bin/kill"
```

Comment:
Yeah, I saw that; I really meant "works", not "syntax-checks".  but I assume you tested it; I'm just saying "sudoers:  O_o"